src/os/CMakeLists.txt

```diff
@@ -101,7 +101,7 @@ if(WITH_BLUESTORE)
   add_executable(ceph-bluestore-tool
     bluestore/bluestore_tool.cc)
   target_link_libraries(ceph-bluestore-tool
-    os global)
+    $<$<PLATFORM_ID:Windows>:ceph-common> os global)
```

Comment:
global is already linked against ceph-common. and per the CMake document

> `target_link_libraries(<target> <item>...)`
> Library dependencies are transitive by default with this signature. When this target is linked into another target then the libraries linked to this target will appear on the link line for the other target too. 

ceph-common is pulled in as part of `global` linkage, when the latter is linked against `ceph-bluestore-tool`. so i don't think this change is necessary.